app/assets/javascripts/sufia/save_work/uploaded_files.es6

```diff
@@ -2,6 +2,8 @@ export class UploadedFiles {
   // Monitors the form and runs the callback if any files are added
   constructor(form, callback) {
     this.form = form
+    this.fileRequirement = form.find('li#required-files')
+    this.hasFileRequirement = this.fileRequirement.size() > 0
```

Comment:
I think you could make `hasFileRequirement` a separate method which returns true or false, and then test it via Jasmine. It would look something like:
``` javascript
it("returns true", function() {
  up = new UploadFiles();
  loadFixtures("html_with_selector.html");
  expect(up.hasFileRequirement).toBe(true);
});